huaweicloud/resource_huaweicloud_compute_volume_attach.go

```diff
@@ -104,20 +107,17 @@ func resourceComputeVolumeAttachV2Create(d *schema.ResourceData, meta interface{
 		return fmtp.Errorf("Error attaching HuaweiCloud volume: %s", err)
 	}
 
-	logp.Printf("[DEBUG] Created volume attachment: %#v", attachment)
-
-	// Use the instance ID and attachment ID as the resource ID.
-	// This is because an attachment cannot be retrieved just by its ID alone.
-	// attachment ID equals the volume ID
-	id := fmt.Sprintf("%s/%s", instanceId, attachment.ID)
+	// The old logic use the instance ID and attachment ID as the resource ID, and the attachment ID equals
+	// the volume ID.
+	id := fmt.Sprintf("%s/%s", instanceId, volumeId)
 	d.SetId(id)
 
-	return resourceComputeVolumeAttachV2Read(d, meta)
+	return resourceComputeVolumeAttachRead(d, meta)
 }
 
-func resourceComputeVolumeAttachV2Read(d *schema.ResourceData, meta interface{}) error {
+func resourceComputeVolumeAttachRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*config.Config)
-	computeClient, err := config.ComputeV2Client(GetRegion(d, config))
+	computeClient, err := config.ComputeV1Client(GetRegion(d, config))
```

Comment:
declare a region may be better , can reused in line 141

Reply:
done